src/main/java/uk/gov/hmcts/reform/idam/client/models/GeneratePinRequest.java

```diff
@@ -1,26 +1,21 @@
 package uk.gov.hmcts.reform.idam.client.models;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
-import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.RequiredArgsConstructor;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Getter
 @Builder
-@AllArgsConstructor
+@RequiredArgsConstructor
 public class GeneratePinRequest {
-
     private final String firstName;
     @JsonInclude
     private final String lastName;
-    private final List<String> roles;
 
-    public GeneratePinRequest(String name) {
```

Comment:
this is a breaking change, the existing constructor should likely be kept for existing users?